### PR TITLE
April Fools, Take Two

### PR DESCRIPTION
**Pull request by Ivniinvi**
## About The Pull Request
Ike didn't like the code or some of the jokes from my first PR, so I've taken my two favorite, small jokes and opened this PR.

## Changelog
:cl:
tweak: The AI Eye now has legs and is visible
tweak: The Gulag Teleporter now sends you to Acacia (no player facing changes)
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/prisoner/gulag_teleporter.dm [hunk 1]
```diff
@@ -148,11 +148,12 @@
...
-		prisoner.Paralyze(40) // small travel dizziness
-		to_chat(prisoner, "<span class='warning'>The teleportation makes you a little dizzy.</span>")
-		if(teleporter.locked)
-			teleporter.locked = FALSE
-		teleporter.toggle_open()
-		contained_id = null
-		temporary_record = null
+	var/client/C = prisoner.client
+	if(C)
+		C << link("byond://acacia.beestation13.com:7979")
```
Comment (ike709) [inline code/game/machinery/computer/prisoner/gulag_teleporter.dm]: IMO if you're on Acacia it should send you to Austation.

**Review by ike709 — CHANGES_REQUESTED** [review]
AI camera needs a better name/desc, but otherwise this is a decently funny PR.

**Comment by PowerfulBacon** [comment]
This will definitely not get abused by someone sending all their targets to acacia

**Comment by Ivniinvi** [comment]
> This will definitely not get abused by someone sending all their targets to acacia

I mean, they could also just, round remove their target normally? Debrain them and stick it in a pot plant? This machine is only in the brig, and this method will only be available for 24h, and it's a funny joke that people will get a kick out of.

**Pull request closed without merging**